spec/features/photos_feature_spec.rb

```diff
@@ -0,0 +1,31 @@
+require 'rails_helper'
+
+feature 'photos' do
+  context 'no photos have been added' do
+    before do
+      visit '/users/sign_up'
+      fill_in 'Username', with: 'TheMagicalInternationalWoodlouse'
+      fill_in 'Email', with: 'fake@fakeemail.com'
+      fill_in 'Password', with: 'secretone'
+      fill_in 'Password confirmation', with: 'secretone'
+      click_button 'Sign up'
+    end
+
+    scenario 'should display a prompt to upload a photo' do
+      visit '/photos'
```

Comment:
Do we need to visit this page? I feel like this appears when we've just signed up anyhoo